mmv1/third_party/terraform/services/bigtable/resource_bigtable_authorized_view.go

```diff
@@ -0,0 +1,462 @@
+package bigtable
```

Comment:
Documentation file needs to be added: https://googlecloudplatform.github.io/magic-modules/develop/handwritten-docs-style-guide/

Reply:
Done.